app/src/org/commcare/preferences/HiddenPreferences.java

```diff
@@ -28,7 +29,7 @@
     public final static String POST_UPDATE_SYNC_NEEDED = "post-update-sync-needed";
     public final static String AUTO_UPDATE_IN_PROGRESS = "cc-trying-to-auto-update";
     public final static String LAST_UPDATE_ATTEMPT = "cc-last_up";
-    public final static String LAST_SYNC_ATTEMPT = "last-ota-restore";
+    public final static String LAST_UPLOAD_SYNC_ATTEMPT = "last-upload-sync";
```

Comment:
Small point: I'm not sure it's worth it to change the *value* of this key (compared to just the static variable name). It makes things more confusing, but changing the value will wipe any existing values for existing users.

In this case that's probably not a big deal, because anyone using this flag has autosync set up at max once a week, but worth considering user impact from the change.

Reply:
Hey agree with the concept in general though in this case we are also making a change where we are storing this info per user per app rather than just per app. And therefore the actual value is stored in a key "userid + "_" +  preferenceKey" . We can write a logic where it checks for the old preference key in case it doesn't find the new one but I am not sure if it's worth the lines of code that are going to stay in the codebase forever.

Reply:
Aha, got it. I noticed the per-user part but failed to realize it was new.

Agree with you that it's not worth it to muddy the codebase based on that, and that a clean start is warranted